Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams_test.py

```diff
@@ -550,7 +560,8 @@ def test_send_message_with_entitlement(mocker, requests_mock):
         'options': ['yes', 'no', 'maybe'],
         'entitlement': '4404dae8-2d45-46bd-85fa-64779c12abe8',
         'investigation_id': '72',
-        'task_id': '23'
+        'task_id': '23',
+        'form_type': 'predefined-options'
```

Comment:
why this change?

Reply:
it was missing this required key in the ask message, (this key was added a couple of months ago, so i guess the test was not updated), but when i added the validation to the ask keys, it failed